src/components/views/rooms/RoomList.tsx

```diff
@@ -387,6 +388,12 @@ const TAG_AESTHETICS: TagAestheticsMap = {
         defaultHidden: false,
         AuxButtonComponent: DmAuxButton,
     },
+    [DefaultTagID.Conference]: {
+        sectionLabel: _td("voip|metaspace_video_rooms|conference_room_section"),
+        isInvite: false,
+        defaultHidden: false,
+        AuxButtonComponent: DmAuxButton,
```

Comment:
This seems unexpected, given a DM won't be added to the Conferences sublist

Reply:
Thanks!